ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ENLIGHTED_FLOOR:
-  id: "4890265568705249280"
-  description: "A basis entity for all enlighted entities."
+
+EXIT:
```

Comment:
Wonder if it makes more sense to have a Fire Life Safety namespace and put this there. These aren't really lights; they have a light in them, but they aren't primarily for illuminating things. Thoughts?

Reply:
They do have a light that we might want to monitor (how many hours has it been on, is it on or off, etc).
There are also emergency luminaires (ligths) that do illuminate and also have a battery, but I was thinking of modelling these using the LT entity type and adding emergency datapoints / fields / subfields.